golang/pkg/dagent/utils/volume.go

```diff
@@ -16,3 +16,15 @@ func MountListToMap(in []mount.Mount) map[string]mount.Mount {
 
 	return mountMap
 }
+
+func FindVolumeInMountMap(name string, mountMap map[string]mount.Mount) *mount.Mount {
+	volumeNameSplit := strings.Split(name, "/")
+	volumeName := volumeNameSplit[len(volumeNameSplit)-1]
```

Comment:
What if the name parameter is an empty string? Just asking.

Reply:
Volume names are validated on the UI so it shouldn't be possible.